scio-avro/src/main/scala/com/spotify/scio/avro/syntax/SCollectionSyntax.scala

```diff
@@ -96,7 +96,7 @@ final class ObjectFileSCollectionOps[T](private val self: SCollection[T]) extend
       shardNameTemplate,
       tempDirectory
     )
-    self.write(ObjectFileIO(path))(param)
+    self.write(ObjectFileIO(path)(coder))(param)
```

Comment:
wasn't the coder taken implicitly ?

Reply:
Thanks for the reminder to look into this. `ObjectFileIO` was missing its type param and getting confused by the new coders in scope from the `avro._` import. Fixed.